setup.py

```diff
@@ -26,12 +28,13 @@
     install_requires=[
         'numpy >= 1.15.0',
         'scipy >= 1.0.0', 
-        'pandas',
-        'matplotlib'
+        'pandas >= 1.0.0',
+        'matplotlib >= 3.3.0'
     ],
     extras_require={
         'docs': [
             'sphinx != 1.3.1',
+            'nbsphinx',
```

Comment:
I think I made changes in the same place in my PR, will this potentially cause a little merge conflict? (I don't think it will be very serious though)

Reply:
The rest looks good to me!

Reply:
@thuntran if there arises one, I'll solve it :)